script/openqa-bootstrap

```diff
@@ -68,24 +72,26 @@ if ping -c1 download.suse.de. && (! rpm -q ca-certificates-suse) ; then
 fi
 
 # fetch tests and needles
-if ping -c1 gitlab.suse.de. ; then
-    # use faster local mirror if run from within SUSE network
-    export needles_giturl="https://gitlab.suse.de/openqa/os-autoinst-needles-opensuse-mirror.git"
-fi
-/usr/share/openqa/script/fetchneedles
```

Comment:
And suddenly, fetchneedles is not called for opensuse (ie. `$skip_suse_tests` not set), see https://progress.opensuse.org/issues/130201

Reply:
I read the "And suddenly" as "this is an obvious mistake". So are you going to prepare a PR to execute this non-SUSE-specific unconditionally?

Although I'm wondering where `skip_suse_tests` is actually set to a non-empty value.